lib/db.js

```diff
@@ -585,11 +643,23 @@ Db.prototype.eval = deprecate(function(code, parameters, options, callback) {
  * @param {Db~collectionResultCallback} [callback] The results callback
  * @return {Promise} returns Promise if no callback passed
  */
+const renameCollectionSchema = {
+  dropTarget: { type: 'boolean', default: false },
+  session: { type: ClientSession },
+  w: { type: ['number', 'string'] },
+  wtimeout: { type: 'number' },
+  j: { type: 'boolean' },
+  new_collection: { overrideOnly: true }
```

Comment:
I believe when I was investigating this in the spring, I found that `new_collection` is actually not used anymore. See https://jira.mongodb.org/browse/SPEC-1054. We should probably remove it.